nemo/core/classes/exportable.py

```diff
@@ -63,6 +63,60 @@ def get_format(filename: str):
         except KeyError:
             raise ValueError(f"Export file {filename} extension does not correspond to any export format!")
 
+    @property
+    def input_module(self):
+        return self
+
+    @property
+    def output_module(self):
+        return self
+
+    def get_input_names(self, input_example):
+        if isinstance(input_example, Dict):
+            input_names = list(input_example.keys())
+        else:
+            if not (hasattr(self, 'input_types')):
+                raise NotImplementedError(
+                    'For export to work you must define input_types or pass names in input_example'
+                )
+            input_names = list(self.input_types.keys())
+        # remove unnecessary inputs for input_ports
+        for name in self.disabled_deployment_input_names:
+            input_names.remove(name)
+        return input_names
+
+    def get_output_names(self, output_example):
+        if isinstance(output_example, Dict):
+            output_names = list(output_example.keys())
+        else:
+            if not (hasattr(self, 'output_types')):
+                raise NotImplementedError(
+                    'For export to work you must define output_types or pass names in output_example'
+                )
+            output_names = list(self.output_types.keys())
+            # remove unnecessary inputs for input_ports
+        for name in self.disabled_deployment_output_names:
```

Comment:
Same comments as above - maybe needs a hasattr(self, ...) first ?

Reply:
Why, it's there in Exportable ?

Reply:
Oh I missed that. LGTM then.